types/ioredis/index.d.ts

```diff
@@ -57,11 +57,33 @@ declare namespace IORedis {
 
     type ValueType = string | Buffer | number | any[];
 
-    interface Command {
+    interface CommandOptions {
+        /**
+         * Set the encoding of the reply, by default buffer will be returned.
+         */
+        replyEncoding?: string | null;
+        errorStack?: string;
+        keyPrefix?: string;
+    }
+
+    interface CommandStatic {
+        new(
```

Comment:
At this point making it a class seems more reasonable.

Reply:
Done.